Tetris/Game.cs

```diff
@@ -16,10 +16,27 @@ public class Game {
         public void draw(Graphics g) {
             g.FillRectangle(new SolidBrush(Constants.BACKGROUND_COLOR), _view);
 
-            Rectangle modifiedView = _view.addMargin(Constants.GAME_MIN_MARGIN_AREA);
+            Rectangle modifiedView = _view.addMargin(Constants.GAME_MIN_MARGIN_AREA);   //gets the new game area with respect to margins
+
+            //>>>   Calculate Margin Based on Aspect Ratio
+
+            //TODO check over and modify to fit our needs (i think this will work)
+
+            float widthtemp = modifiedView.Width / 5;
+            float heighttemp = modifiedView.Height / 9;
+
+            float lowest = widthtemp < heighttemp ? widthtemp : heighttemp;
+
+            float newwidth = lowest * 5;
+            float newheight = lowest * 9;
+
```

Comment:
Let's put this as an extension method for Rectangle. We should also change the GAME_ASPECT_RATIO constant to two separate constants GAME_ASPECT_HEIGHT and GAME_ASPECT_WIDTH.